packages/worker/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@chainlink/ccip-read-cf-worker",
-    "version": "0.0.1",
+    "version": "0.0.2",
```

Comment:
Didn't you bump to 0.0.3?

Reply:
it's bumped to 0.0.3 for `@ensdomains/ccip-read-cf-worker` package only, `@chainlink/ccip-read-cf-worker` will be `0.0.2` after this merge.